code/game/gamemodes/objective.dm

```diff
@@ -29,41 +41,55 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 		return TARGET_INVALID_IS_TARGET
 	if(!ishuman(possible_target.current))
 		return TARGET_INVALID_NOT_HUMAN
-	if(!possible_target.current.stat == DEAD)
-		return TARGET_INVALID_DEAD
 	if(!possible_target.key)
 		return TARGET_INVALID_NOCKEY
 	if(possible_target.current)
+		if(!possible_target.current.stat == DEAD)
```

Comment:
I think the `!` might break things here. For example `if(!x == 2)` (2 being the value for `DEAD`) equates to 0, which means the if never succeeds, no matter the `stat` value.

Reply:
oh wow. I just copy pasted it without checking really. But yeah... odd how this has been like this for so long. Will fix